content/doc/book/glossary/index.adoc

```diff
@@ -53,7 +53,7 @@ Artifact::
     later retrieval by users.
 Build:: [[build]]
     Result of a single execution of a <<job,job>>
-Cloud:: [[cloud]]
+https://www.jenkins.io/doc/developer/extensions/alibabacloud-ecs/#ji-toolbar[Cloud]:: [[cloud]]
```

Comment:
This link won't help the reader when they click it.  Unfortunately, I don't find a good high level description of cloud agents or ephemeral agents on www.jenkins.io.  We'll need to add that before this link can be created.
```suggestion
Cloud:: [[cloud]]
```